yaksh/templates/yaksh/course_detail_options.html

```diff
@@ -0,0 +1,47 @@
+<ul class="nav nav-pills list-group">
+    <li class="nav-item">
+        <a href="{% url 'yaksh:courses' %}" class="nav-link list-group-item" title="View all the courses" data-placement="top" data-toggle="tooltip">
```

Comment:
Tooltip is distracting. Please remove if possible, or decrease its disappearing animation time.